Source/WebKit/WebProcess/GPU/graphics/RemoteImageBufferProxy.cpp

```diff
@@ -430,6 +320,12 @@ void RemoteImageBufferProxy::prepareForBackingStoreChange()
         backend->ensureNativeImagesHaveCopiedBackingStore();
 }
 
+void RemoteImageBufferProxy::remoteBufferWasDestroyed()
+{
+    if (m_remoteRenderingBackendProxy)
+        m_remoteRenderingBackendProxy->createRemoteImageBuffer(*this);
```

Comment:
Is this change still related to this particular fix or some other fix?

Reply:
Leftover from previous changes, removed.